docs/search_dsl.rst

```diff
@@ -391,7 +391,7 @@ To specify a suggest request on your ``Search`` object use the ``suggest`` metho
 
 .. code:: python
 
-    s = s.suggest('my_suggestion', 'pyhton', term={'field': 'title'})
+    s = s.suggest('my_suggestion', 'python', term={'field': 'title'})
```

Comment:
this misspelling has actually been on purpose to show the typical use case for suggestions - the "did you mean?" spell checker :)